scripts/contrib/completion/bash/netctl

```diff
@@ -1,11 +1,19 @@
-#! /bin/bash
+#!/bin/bash
 
 GLOBAL_COMMANDS="\
+    version\
     group\
+    endpoint\
+    netprofile\
     network\
     tenant\
     policy\
+    external-contracts\
     global\
+    aci-gw\
+    bgp\
+    app-profile\
+    service\
     help"
```

Comment:
Should we re-order bash/netctl and netctl/commands.go in a separate fix after this to keep this diff readable?